packages/actor-rdf-resolve-quad-pattern-string-source/lib/ActorRdfResolveQuadPatternStringSource.ts

```diff
@@ -37,9 +41,26 @@ export class ActorRdfResolveQuadPatternStringSource extends ActorRdfResolveQuadP
     return true;
   }
 
-  public async run(action: IActionRdfResolveQuadPattern): Promise<IActorRdfResolveQuadPatternOutput> {
-    const source = <IDataSourceSerialized>getContextSource(action.context);
+  public run(action: IActionRdfResolveQuadPattern): Promise<IActorRdfResolveQuadPatternOutput> {
+    const source = <IDataSourceSerialized>getContextSource(action.context)!;
```

Comment:
Could you also add a small jsdoc comment explaining why exactly we need to cache the promise?

Reply:
I added a comment to tge getRdfSource function, as well as a short note just above the line where the first cache get is located. Do they look okay, or should they be moved somewhere or adjusted?